src/helpers/node-discovery.ts

```
@@ -0,0 +1,1212 @@
+      return 'Data Transformation';
+    }
+    
+    return 'Regular Nodes';
+  }
+
+  /**
+   * Check if a node is a trigger node
+   */
+  private isTriggerNode(apiNode: any): boolean {
+    return apiNode.trigger === true ||
+           (apiNode.name && apiNode.name.toLowerCase().includes('trigger')) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)